sdk/resourcemanager/postgresql/configuration/README.md

```diff
@@ -5,7 +5,7 @@ languages:
 products:
 - azure
 description: "These code samples will show you how to manage Database for PostgreSQL using Azure SDK for Golang."
-urlFragment: postgresql
+urlFragment: postgre-sql-config
```

Comment:
Should these be `postgresql-<whatever>`?

Reply:
They can be whatever value you prefer -- but the samples in the postgresql path cannot have the same URL fragment. (The pattern postgresql-<whatever> should be okay too)

Reply:
Can we do the same for postgresql so the paths align?

Reply:
You would like the pattern to be postgresql-<whatever>?

Reply:
Yeah please.